tests/remote.rs

```diff
@@ -101,7 +102,11 @@ async fn raw_inner(mut c: Client) -> Result<(), error::CmdError> {
     let src = img.attr("src").await?.expect("image should have a src");
 
     // now build a raw HTTP client request (which also has all current cookies)
-    let raw = img.client().raw_client_for(Method::GET, &src).await?;
+    let raw = img
+        .client()
+        .clone()
```

Comment:
I'm not sure I follow why this now requires a `clone`?

Reply:
It doesn't. It's leftover from a previous change.